### PR TITLE
update note about cobbler on SMR to reflect TFTP

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/SUSE/spacewalk/issues/10241

I hope I understood this correctly.

### CODE REVIEW

File: modules/salt/pages/formula-tftpd.adoc [hunk 1]
```diff
@@ -1,7 +1,13 @@
 [[tftpd-formula]]
 = TFTPd Formula
 
-The TFTPd formula is used to configure the TFTP service on the branch server.
+The TFTPd formula is used to configure the TFTP service on the {productname} Proxy.
+
```
Comment (keichwa) [inline modules/salt/pages/formula-tftpd.adoc]: "TFTPd formula" vs. "`tftpd` formula" spelling.  I'd uppercase it erverywhere (TFTPD) or use two words (TFTP daemon). Software to be changed accordingly, IMO.

**Comment by aaannz** [comment]
As @mcalmer said. TFTP is required by both cobbler and retail, however they can't work together at the same time on the branch proxy (and in the future in some cases also on suse manager).
So I would:

* in the cobbler.adoc replace the paragraph with something like 'Cobbler installation server is not supported within {smr} environments. If you intend to use your installation with {smr} formulas, do not configure Cobbler TFTP on a branch server`.
* in the retail-install-unified.adoc replace the `Cobbler` with `susemanager-tftpsync-recv`
* do not add anything in the tftp formula. Tftp formula is specifically used by retail, it does not make any sense to not be supported on the branch server

**Comment by mcalmer** [comment]
> * in the cobbler.adoc replace the paragraph with something like 'Cobbler installation server is not supported within {smr} environments. If you intend to use your installation with {smr} formulas, do not configure Cobbler TFTP on a branch server`.

We cannot prevent "cobbler". Cobbler exists on the "Server" only and is a hard required component. We better talk about tftpsync:

'Tftpsync is not supported within {smr} environments. If you intend to use your installation with {smr} formulas, do not tftpsync on a branch server`.

**Comment by aaannz** [comment]
> We cannot prevent "cobbler". Cobbler exists on the "Server" only and is a hard required component. We better talk about tftpsync:

I understand that. What I try to explain is that retail user should skip that whole page. Because once we enable branchless retail scenario, suma itself is being managed as minion and retail tftpd formula will change existing tftpd configuration.

**Comment by Loquacity** [comment]
@keichwa can you please confirm that I fixed that merge conflict OK?

**Comment by keichwa** [comment]
Lana Brindley <notifications@github.com> writes:

> @keichwa can you please confirm that I fixed that merge conflict OK?

Yes, well done!

-- 
Karl Eichwalder            SUSE Software Solutions Germany GmbH
PES / Documentation        Maxfeldstraße 5
                           90409 Nürnberg, Germany
Geschäftsführer: Felix Imendörffer (HRB 36809, AG Nürnberg)

**Pull request merged**